### ISSUE TITLE
[1.15.2] Bug Report (Rifters Move In Circles)

### ISSUE THREAD

**Issue by M240306** [Cryptic-Mushroom/The-Midnight#433]
Minecraft 1.15.2
Forge 31.2.41
Rifters move in circles
Hello I am having a bug with rifters that when they grab you they move in circles instead of carring you to the portal. I dont know if its like this but in older versions the rifter carried you to the portal. 

Here is a video of the bug: 
https://drive.google.com/file/d/1pDhyW250Ob3FVhmAdRWW88-_37Tf8ORU/view

I hope you could fix it.

Thank you, 
M240306

**Comment by ShadewRG** [comment]
We need to request access for the video...

**Comment by M240306** [comment]
ok wait one minute

**Comment by M240306** [comment]
Now can you see it?

**Comment by M240306** [comment]
https://drive.google.com/file/d/1pDhyW250Ob3FVhmAdRWW88-_37Tf8ORU/view?usp=sharing

**Comment by Jonathing** [comment]
Yup, we can see it now. This has been a known issue for a while because the Rifters broke when the mod was ported to 1.15.2. We're rewriting rifts and rifters from scratch next update so this won't be an issue, but it's unlikely we'll be patching this in the current release right now.

**Comment by M240306** [comment]
Ok thank you

**Comment by M240306** [comment]
But can you upload like a version of 1.14.4 with the old rifts and rifters compatible with 1.15.2? Just asking because I dont think is posible :)

**Comment by Jonathing** [comment]
We are no longer working on the current version of The Midnight. Although development will be slow, we are working on a rewrite of the mod which will feature both permanent rifts as structures and temporary rifts like the ones in 1.14.4.

**Comment by M240306** [comment]
ok, and your discord still open?, because I try to enter and it says not valid.

**Comment by Jonathing** [comment]
We've been on lockdown for a little bit because of some bot raids, but later today I'll update the Discord links on our pages.

**Comment by M240306** [comment]
ok thanks

**Comment by Jonathing** [comment]
Fixed in 0.5.10. This issue will stay open to keep this in mind for the 0.6.0 update.
